src/pybind/mgr/dashboard/cherrypy_backports.py

```diff
@@ -75,7 +75,7 @@ def patch_builtin_ssl_wrap(v, new_wrap):
     if v < StrictVersion("9.0.0"):
         from cherrypy.wsgiserver.ssl_builtin import BuiltinSSLAdapter as builtin_ssl
     else:
-        from cherrypy.cheroot.ssl.builtin import BuiltinSSLAdapter as builtin_ssl
```

Comment:
Hm. Was there a timeframe, where `cherook` was already separate (extracted from wsgiserver), but still was part of `cherrypy`?

Reply:
looking at the version diff of v8.9.1..v9.0.0 (there are no versions in-between), I can see the hard cut for cheroot.
While `BuiltinSSLAdapter` is not present in 9.0.0, it still is in 8.9.1.

Reply:
> `cherook`

I assume you're working too much with rook. 😂